util/missingcrypto.txt

```diff
@@ -1120,8 +1123,10 @@ SHA256_Transform(3)
 SHA512_Transform(3)
 SMIME_crlf_copy(3)
 SMIME_read_ASN1(3)
+SMIME_read_ASN1_ex(3)
 SMIME_text(3)
 SMIME_write_ASN1(3)
+SMIME_write_ASN1_with_libctx(3)
```

Comment:
Why are we adding new undocumented functions?

Reply:
Because it is derived from an existing undocumented function.

Reply:
So lets document it!

Reply:
I was afraid you might say that - now I have to figure out what they really do..

Reply:
Ok added docs now.